x-pack/plugin/runtime-fields/src/main/resources/org/elasticsearch/xpack/runtimefields/double_whitelist.txt

```diff
@@ -9,10 +9,6 @@
 class org.elasticsearch.xpack.runtimefields.DoubleScriptFieldScript @no_import {
 }
 
-static_import {
-    void value(org.elasticsearch.xpack.runtimefields.DoubleScriptFieldScript, double) bound_to org.elasticsearch.xpack.runtimefields.DoubleScriptFieldScript$Value
-}
-
-# This import is required to make painless happy and it isn't 100% clear why
+# This whitelist is required to allow painless to build the factory
 class org.elasticsearch.xpack.runtimefields.DoubleScriptFieldScript$Factory @no_import {
 }
```

Comment:
I was hoping that this file would go away completely. can you remind what the remaining lines are for?

Reply:
I'll try one more time to drop it. I tried and it blew up. But I'll try without it entirely.

I *think* these are required so painless can use the classes that we defined. But these might should all be "implied" because they are part of the script context.

Reply:
I checked - it is still required.

Reply:
@stu-elastic and @jdconrad  - I think we still need this file because without it painless can't find the factory class. I *think* that is because is part of a plugin and not in painless's classloader. Or something like that. Is that how the world is supposed to be? Could these classes be implied by default because the context mentions them directly?

Reply:
Painless has no way to load these classes w/o explicitly stating them as part of SPI.

Reply:
:+1:.

Could you dig them out of the `ScriptContext`, I wonder?

Reply:
That's something worth looking into. Agreed.